lib/Doctrine/ODM/PHPCR/Mapping/ClassMetadataFactory.php

```diff
@@ -256,6 +256,7 @@ protected function validateRuntimeMetadata($class, $parent)
         $class->validateIdentifier();
         $class->validateReferenceable();
         $class->validateReferences();
+        $class->validateChildClasses();
```

Comment:
kind of strange that we don't have just one validate method on ClassMetadata and call that here, and keep the details of what needs to be validated local to ClassMetadata...

but not subject of this PR, so lets keep it like this here.